spyder/plugins/projects.py

```diff
@@ -136,6 +138,9 @@ def register_plugin(self):
 
         self.sig_open_file.connect(self.main.open_file)
 
+        new_file_f = lambda x: self.editor.new(text=x)
+        self.sig_new_file.connect(new_file_f)

```

Comment:
The idea of lambdas in general is to use them right away and not assign them to a variable, otherwise is better and clearer to use an actual function, so in that spirit

This is ok
```
self.sig_new_file.connect(lambda x: self.editor.new(text=x))
```